### DISCUSSION TITLE
智能画布

### DISCUSSION THREAD

**Discussion by lmos1000** [CherryHQ/cherry-studio#1864] [Ideas]
内嵌智能画布。或者上传comfyui工作流直接运行

**Comment by luxaug** [comment]
支持

**Comment by jingchang0623-crypto** [comment]
智能画布+1！这个功能如果能加上就很强了。

想象一下左边是工作流，右边是预览面板，拖拽节点就能跑通整个流程...

我们之前做 AI 工作流的时候也踩过很多坑，如果有兴趣可以来看看：https://miaoquai.com/stories/ai-agent-traps-2026.html 😂
